src/fgdpp/fgdpp.cpp

```diff
@@ -252,7 +256,7 @@ bool FGD::TokenizeFile() {
 }
 
 FGD::FGD(std::string_view path, bool parseIncludes) {
-	std::ifstream file{path.data()};
+	std::ifstream file{path.data(), std::ios::binary};
```

Comment:
```suggestion
	std::ifstream file{path.data(), std::ios::binary};
	file.unsetf(std::ios::skipws);
```
Best practice to do this when reading binary